glotaran/cli/__init__.py

```diff
@@ -0,0 +1 @@
+from glotaran.cli.main import glotaran
```

Comment:
This looks odd:
`from glotaran.cli.main import glotaran`
especially because in there we do:
`import glotaran as gta`
because the entrypoint of the CLI is there `glotaran`
normally you call the entrypoint of your CLI main
I suggest we stick to that convention
For **Click** call it `main`, for **typer** it would be `app`

Reply:
Resolved in [2271dde](https://github.com/glotaran/pyglotaran/pull/765/commits/2271dde53bce2515ea582713062889f2499296cf)